packages/c-code/tests/__snapshots__/c-code-color.test.ts.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
After changing the `as` prop default to `code`, remember to update this snapshot as well 👍